tpgtools/types/resource.go

```diff
@@ -306,7 +305,7 @@ func (r Resource) RegisterReusedType(p Property) []Property {
 	return r.ReusedTypes
 }
 
-func createResource(schema *openapi.Schema, typeFetcher *TypeFetcher, overrides Overrides, packagePath string, location string) (*Resource, error) {
+func NewResource(schema *openapi.Schema, typeFetcher *TypeFetcher, overrides Overrides, packagePath string, location string) (*Resource, error) {
```

Comment:
nit: Since this is exported now, we'll want to comment it

Reply:
will add